packages/live-share-canvas/src/core/LiveCanvas.ts

```diff
@@ -509,11 +518,11 @@ export class LiveCanvas extends DataObject {
             scope,
             InkingEventNames.pointerMove,
             (evt: IPointerMovedEvent, local: boolean) => {
-                if (!local && evt.clientId) {
+                const clientId = evt.clientId;
```

Comment:
Why this change? That's an unnecessary constant... I could understand if it was like `someLongVarName.eventLonger.somethingElse.clientId`, but `evt.clientId` is just as clear as `clientId`